### PR TITLE
HAWQ-39. Remove below unused mgmt scritps for hawq2.0:

### PR DESCRIPTION
**Pull request by radarwave**
gpactivatestandby gpinitstandby gpinitsystem gpseginstall lib/gpcreateseg.sh lib/gpinitsegment lib/gpseginitsb.sh

### CODE REVIEW

**Comment by huor** [comment]
The removal of the obsolete HAWQ management scripts seems good. +1

**Comment by yaoj2** [comment]
Looks good to me since we have new hawq command.

**Comment by nellaivijay** [comment]
I have gone thru gpactivatestandby gpinitstandby gpinitsystem gpseginstall lib/gpcreateseg.sh lib/gpinitsegment lib/gpseginitsb.sh scripts. Its good to remove from current package. +1

File: tools/Makefile [hunk 1]
```diff
@@ -22,7 +22,6 @@ clean distclean:
 #---------------------------------------------------------------------
 
 SET_VERSION_SCRIPTS = \
-	bin/gpactivatestandby \
 	bin/gpaddmirrors \
```
Comment (cwelton) [inline tools/Makefile]: Why leave gpaddmirrors?  HAWQ doesn't have mirrors to begin with (legacy from Greenplum database).

File: tools/Makefile [hunk 2]
```diff
@@ -33,8 +32,6 @@ SET_VERSION_SCRIPTS = \
 	bin/gpexpand \
```
Comment (cwelton) [inline tools/Makefile]: Why leave gpexpand?  This has been replaced with the HAWQ 2.0 mechanisms around adding/removing nodes.

**Comment by cwelton** [comment]
The changes present look good.  Mostly the questions I have are around the things that are left, gpaddmirrors, gpexpand, gpstart, gpstop, gpstate, gprecoverseg, gpmigrator_mirror, gpmovemirrors.

**Comment by radarwave** [comment]
Thanks all for the review. The other legacy scripts will be remove batch by batch. Just be careful if it might cause some problem.

**Pull request merged**